connector-email/src/main/resources/application.properties

```diff
@@ -0,0 +1,52 @@
+notifications.connector.endpoint-cache-max-size=100
+notifications.connector.kafka.incoming.group-id=notifications-connector-email
+notifications.connector.kafka.incoming.topic=${mp.messaging.tocamel.topic}
+notifications.connector.kafka.outgoing.topic=${mp.messaging.fromcamel.topic}
+notifications.connector.name=email_subscription
+notifications.connector.redelivery.counter-name=camel.email.retry.counter
+notifications.connector.redelivery.delay=1000
+notifications.connector.redelivery.max-attempts=2
+
+quarkus.http.port=9003
+
+quarkus.log.cloudwatch.enabled=false
+quarkus.log.cloudwatch.level=INFO
+quarkus.log.cloudwatch.log-stream-name=notifications-connector-email
+
+quarkus.log.sentry=false
+quarkus.log.sentry.in-app-packages=com.redhat.cloud.notifications
+
+quarkus.kafka.devservices.port=9092
+
+camel.component.kafka.brokers=localhost:9092
+camel.component.kafka.sasl-jaas-config=""
+camel.component.kafka.sasl-mechanism=GSSAPI
+camel.component.kafka.security-protocol=PLAINTEXT
+camel.component.kafka.ssl-truststore-location=
+camel.component.kafka.ssl-truststore-type=JKS
+camel.component.kafka.poll-on-error=RECONNECT
+camel.component.kafka.retries=3
+camel.component.kafka.retry-backoff-ms=200
+
+camel.context.name=notifications-connector-email
+
+mp.messaging.tocamel.topic=platform.notifications.tocamel
+mp.messaging.fromcamel.topic=platform.notifications.fromcamel
+
+%test.quarkus.log.category."com.redhat.cloud.notifications".level=DEBUG
+
+# The "no reply" address used to set as the sender in the emails the connector
+# sends.
+email.no-reply.address=noreply@redhat.com
+
+# Configuration of the various user providers that the connector supports.
+user-provider.bop.api_token=changeme
+user-provider.bop.client_id=changeme
+user-provider.bop.env=changeme
+user-provider.bop.url=http://boplocalhost:9999
+user-provider.it.url=http://itlocalhost:9999
+user-provider.it.elements-per-page=5
+user-provider.rbac.url=http://rbaclocalhost:9999
+#user-provider.rbac.url=http://localhost:8080
+user-provider.rbac.elements-per-page=5
```

Comment:
Let's try to start all config keys from connectors with `notifications.connector.`.

I also introduced many changes (with inline comments only for this review, to be removed) in the suggestion below. Some of them are purely a question of personal preference so you don't have to apply everything :smiley: 

```suggestion
# The "no reply" address used to set as the sender in the emails the connector
# sends.
notifications.connector.no-reply-email-address=noreply@redhat.com # FYI this will likely go away soon, I'm not even sure we need it in this PR

# Configuration of the various user providers that the connector supports.

notifications.connector.bop.apitoken=changeme # matches the header name sent to BOP
notifications.connector.bop.clientid=changeme # same
notifications.connector.bop.env=changeme
notifications.connector.bop.url=http://boplocalhost:9999

notifications.connector.it-users-service.url=http://itlocalhost:9999
notifications.connector.it-users-service.elements-per-page=5

notifications.connector.rbac.url=http://rbaclocalhost:9999
notifications.connector.rbac.elements-per-page=5
```

Reply:
Done as well!